vars/gradleWrapper.groovy

```diff
@@ -1,18 +1,49 @@
 #!/usr/bin/env groovy
 
-def cleanupCommand(String command) {
+/**
+ * @param command
+ * @return A formatted command for the gradle process. If a log level was provided by the
+ * environment it will be appended to the command.
+ */
+def formatCommand(String command) {
    regex = /-P[\w\d.]+=\s/
-   command.replaceAll(regex, '')
+   def result = command.replaceAll(regex, '')
+
+    if (!containsLogLevel(result)) {
+        if (params.LOG_LEVEL){
+            result += " --${params.LOG_LEVEL}"
+        }
+        else if (env.LOG_LEVEL) {
+            result += " --${env.LOG_LEVEL}"
+        }
+    }
+
+    if (params.STACK_TRACE == true) {
+        result += " --stacktrace"
+    }
```

Comment:
We would also need to check if the `--stacktrace` option is provided. Just so the command is cleaner

Reply:
Fair enough; I had avoided doing so since I assumed teams wouldn't commit the --stacktrace option to the main branch.